nextcord/ext/commands/bot.py

```diff
@@ -67,7 +68,7 @@
 CXT = TypeVar("CXT", bound="Context")
 
 
-def when_mentioned(bot: Union[Bot, AutoShardedBot], msg: Message) -> List[str]:
+def when_mentioned(bot: Union[Bot, AutoShardedBot], _msg: Message) -> List[str]:
```

Comment:
It *is* unused here, but custom prefixes can make use of that message to have per-guild prefixes for example. It is a breaking change though yeah since you can use this function within a custom prefix function.